include/session/config/encrypt.hpp

```diff
@@ -62,6 +62,9 @@ void encrypt_inplace(
 ///
 /// That is, for some message `m`, encrypt_overhead() is the difference between m.size() and
 /// encrypt(m).size().
+///
+/// Inputs: None
+///
```

Comment:
This one is indeed a parser problem; I've added a change in #45 that recognizes this as a constant automatically, but also adds a new `/// Constant.` stanza in cases where the constant might be too complex.